hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/bucket/OMBucketCreateRequest.java

```diff
@@ -153,19 +154,18 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
         getOmRequest());
     OmBucketInfo omBucketInfo = null;
 
-    // bucketInfo.hasBucketLayout() would be true when user sets bucket layout.
-    // Now, OM will create bucket with the user specified bucket layout.
-    // When the value is not specified by the user, OM will use
-    // "ozone.default.bucket.layout" configured value for the newer ozone
-    // client and LEGACY for an older ozone client.
+    // bucketInfo.hasBucketLayout() would be true when the request proto
+    // specifies a bucket layout.
+    // When the value is not specified in the proto, OM will use
+    // "ozone.default.bucket.layout".
     if (!bucketInfo.hasBucketLayout()) {
-      BucketLayout defaultBucketLayout =
-          getDefaultBucketLayout(ozoneManager, volumeName, bucketName);
+      BucketLayout defaultBuckLayout = ozoneManager.getOMDefaultBucketLayout();
```

Comment:
Is it intentional to change the variable name here or accidental? I am fine with this version also it is pretty much up to you, just wanted to note the change so we are aware.